Makefile.vendor.mk

```diff
@@ -0,0 +1,6 @@
+VERSION = 3.0.0
+OPERATOR_NAME = servicemeshoperator3
+HUB = quay.io/maistra-dev
+CHANNELS = 3.0
+HELM_VALUES_FILE = ossm/values.yaml
+VERSIONS_YAML_FILE = versions.yaml
```

Comment:
Do we want to remove versions which we don't build downstream here or in cpaas midstream?

Reply:
same here, I'd say we do that when we branch off into a release branch for tp1